src/components/spec-document/overview/ServerVariables.vue

```diff
@@ -0,0 +1,96 @@
+<template>
+  <div
+    v-if="server.variables"
+    class="variables-container"
+  >
+    <span class="variables-label">Variables</span>
+    <div
+      v-for="key in Object.keys(server.variables)"
+      :key="`${server.id}-${key}`"
+      class="variable-container"
+    >
+      {{ key }}:
+      <SelectDropdown
+        v-if="server.variables[key].enum"
+        :data-testid="`${server.id}-${key}-select`"
+        :items="enumToSelectItem(server.variables[key].enum)"
+        :model-value="server.variables[key].extensions?.value as string || server.variables[key].default"
+        @change="(item) => handleVariableChange(server.id, key, item.value)"
+      />
+      <input
+        v-else
+        :data-testid="`${server.id}-${key}-input`"
+        type="text"
+        :value="server.variables[key].extensions?.value || server.variables[key].default"
+        @change="(e) => handleVariableChange(server.id, key, (e.target as HTMLInputElement).value)"
+      >
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+import type { IServer } from '@/types'
+import type { PropType } from 'vue'
+import SelectDropdown from '@/components/common/SelectDropdown.vue'
+import type { SelectItem } from '@/types'
```

Comment:
```suggestion
import type { IServer, SelectItem } from '@/types'
import type { PropType } from 'vue'
import SelectDropdown from '@/components/common/SelectDropdown.vue'
```

Don't know why eslint doesn't catch these always.